nighthawk/include/nighthawk/client/benchmark_client.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+
+#include "envoy/runtime/runtime.h"
+
+#include "nighthawk/common/statistic.h"
+
+namespace Nighthawk {
+namespace Client {
+
+typedef std::function<bool(const std::string, const uint64_t)> CounterFilter;
+
+class BenchmarkClient {
+public:
+  virtual ~BenchmarkClient() = default;
+
+  /**
+   * Initialize will be called on the worker thread after it has started.
+   * @param runtime to be used during initialization.
```

Comment:
Nit: document `@return`. What would a fail here do?

Reply:
This no longer has a return value